### PR TITLE
Personal and efficient keymap

### PR DESCRIPTION
**Pull request by vennjadin**
## Description

Added my own keymap. It’s a nice one, very efficient

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/vennjadin/rules.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SRC += muse.c
+UNICODE_ENABLE = yes
```
Comment (drashna) [inline keyboards/planck/keymaps/vennjadin/rules.mk:2]: is unicode even being used here?

**Comment by vennjadin** [comment]
Not at this point, but I’m planning to include it later on.

File: keyboards/planck/keymaps/vennjadin/keymap.c [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+),
+
+/* Lower
+ * ,-----------------------------------------------------------------------------------.
+ * |      |   +  |   7  |   8  |   9  |   ,  |   %  | [    |  ]   |  =   |   _  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  -   |  4   |  5   |  6   |  .   |  /   |   (  |  )   | <    |   >  |  '   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   $  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
```
Comment (precondition) [inline keyboards/planck/keymaps/vennjadin/keymap.c:66]: The ASCII art implies <kbd>'</kbd> when in reality, it is `NO_PIPE`, and it also shows <kbd>$</kbd> when in reality it is `NO_CURR`, which is conventionally represented with the symbol ¤.

File: keyboards/planck/keymaps/vennjadin/keymap.c [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+[_LOWER] = LAYOUT_planck_grid(
+    _______, NO_PLUS, KC_7,    KC_8,    KC_9,    KC_COMM, KC_PERC,    NO_LBRC,    NO_RBRC,    NO_EQL,  NO_UNDS, _______,
+    _______, NO_MINS, KC_4,    KC_5,    KC_6,    KC_DOT,  NO_SLSH,    NO_LPRN,    NO_RPRN,    NO_LABK, NO_RABK, NO_PIPE,
+    _______, KC_0,    KC_1,    KC_2,    KC_3,    NO_ASTR, A(NO_SLSH), A(NO_LPRN), A(NO_RPRN), NO_HASH, NO_CURR, _______,
+    KC_ENT,  _______, _______, _______, _______, _______, _______,    _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+/* Raise
+ * ,-----------------------------------------------------------------------------------.
+ * |      |  ~   |  ¨   |   `  |  ´   |   ^  |      | BrD  |  up  |  BrU |      |      |
```
Comment (precondition) [inline keyboards/planck/keymaps/vennjadin/keymap.c:79]:
```suggestion
 * |      |  ~   |  ¨   |   `  |  \   |   ^  |      | BrD  |  up  |  BrU |      |      |
```
The code says it is `NO_BSLS`

File: keyboards/planck/keymaps/vennjadin/readme.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+|      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   $  |      |
+|------+------+------+------+------+------+------+------+------+------+------+------|
+|enter |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+`-----------------------------------------------------------------------------------'
+```
+
+### Raise
+```
+,-----------------------------------------------------------------------------------.
+|      |  ~   |  ¨   |   `  |  ´   |   ^  |      | BrD  |  up  |  BrU |      |      |
```
Comment (precondition) [inline keyboards/planck/keymaps/vennjadin/readme.md:41]:
```suggestion
|      |  ~   |  ¨   |   `  |  \   |   ^  |      | BrD  |  up  |  BrU |      |      |
```

File: keyboards/planck/keymaps/vennjadin/readme.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+ ```
+
+ ### Lower
+ ```
+,-----------------------------------------------------------------------------------.
+|      |   +  |   7  |   8  |   9  |   ,  |   %  | [    |  ]   |  =   |   _  |      |
+|------+------+------+------+------+------+------+------+------+------+------+------|
+|      |  -   |  4   |  5   |  6   |  .   |  /   |   (  |  )   | <    |   >  |  '   |
+|------+------+------+------+------+------+------+------+------+------+------+------|
+|      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   $  |      |
```
Comment (precondition) [inline keyboards/planck/keymaps/vennjadin/readme.md:32]:
```suggestion
|      |  -   |  4   |  5   |  6   |  .   |  /   |   (  |  )   | <    |   >  |  |   |
|------+------+------+------+------+------+------+------+------+------+------+------|
|      |  0   |  1   |  2   |  3   |  *   |  \   |   {  |  }   |  #   |   ¤  |      |
```

**Review by precondition — COMMENTED** [review]
Simply pointed out some inconsequential inconsistencies between what the ASCII art says and what the code says.

**Comment by vennjadin** [comment]
Probably should have pointed this out, but the Norwegian Mac ISO spec is different than the Windows one. So these display as represented in the ASCII. I suggest instead I add this note to the readme (and also might comment it in to the code itself).

File: keyboards/planck/keymaps/vennjadin/keymap.c [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+      #endif
+    } else {
+      #ifdef MOUSEKEY_ENABLE
+        tap_code(KC_MS_WH_UP);
+      #else
+        tap_code(KC_PGUP);
+      #endif
+    }
+  }
+    return true;
```
Comment (drashna) [inline keyboards/planck/keymaps/vennjadin/keymap.c:205]:
```suggestion
    return false;
```

File: keyboards/planck/keymaps/vennjadin/keymap.c [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+  return true;
+}
+
+bool muse_mode = false;
+uint8_t last_muse_note = 0;
+uint16_t muse_counter = 0;
+uint8_t muse_offset = 70;
+uint16_t muse_tempo = 50;
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
```
Comment (drashna) [inline keyboards/planck/keymaps/vennjadin/keymap.c:175]: Also, may want to check out the encoder map feature.

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

**Pull request closed without merging**